uploader.html

```diff
@@ -147,29 +144,29 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
             <!-- selection section container -->
             <div id="js-selected-media-section">
               <script type="text/html">
-                <div id="js-current-label" class="js-current-label col-sm-4">
+                <div id="js-current-label" class="current-label col-sm-4">
                   Current <%= filetype %>
                 </div>
-                <div id="js-current-preview" class="js-current-preview col-sm-4">
+                <div id="js-current-preview" class="current-preview col-sm-4">
                   <img id="js-output-file"/>
                 </div>
                 <div id="js-current-btn" class="col-sm-4">
-                  <button id="js-full-image" class="button btn-sm">View Full Size</button>
+                  <button id="js-full-image" class="btn btn-sm btn-default">View Full Size</button>
                 </div>
               </script>
             </div>
-          </div>
+          </div><!-- close modal-body -->
 
           <div class="modal-footer">
+            <input type="button" value="Close" class="btn btn-sm btn-default" data-dismiss="modal">
```

Comment:
I know that the old code was doing this, but now we're using `<button>` tags instead of `<input type="button" />` because it makes adding translations and icons easier.
```
<button type="button" class="btn btn-sm btn-default data-dismiss="modal">Close</button>
```

Reply:
@biyeun Can you take a look at this?  I'd like to close out this PR this week, if possible.